src/components/AttachmentPicker/index.native.js

```diff
@@ -304,8 +304,9 @@ class AttachmentPicker extends Component {
                 <Popover
                     onClose={this.close}
                     isVisible={this.state.isVisible}
-                    anchorPosition={styles.createMenuPosition}
                     onModalHide={this.onModalHide}
+                    anchorPosition={this.props.anchorPosition}
+                    fromSidebarMediumScreen={!this.props.isSmallScreenWidth}
```

Comment:
How is this prop related to this menu? This menu opens from the composer :heavy_plus_sign: .

Reply:
The menu for attachment picker opens from the same position as the one from the composer ➕.